panda_grasp:
  task_frame_id: task
  base_frame_id: panda_link0
  ee_frame_id: panda_link8
  ee_grasp_offset: [0.0, 0.0, -0.382, 0.924, 0.0, 0.0, 0.065]  # TODO check this
  scan_joints:
    - [0.010, -1.479, 0.10027132190633238, -2.416246868493765, 0.08994288870361117, 1.4022499498261343, 0.8516519819506339]
    - [-0.520, -0.796, 0.9121961832112832, -1.6720657872605567, 0.0914094145960278, 1.2810500415696036, 0.8710614908889162]
    - [0.032, 0.4595, 0.0743635250858064, -0.8394780465249851, 0.01546591704007652, 0.7776030993991428, 0.8335337665490805]
    - [0.092, -0.626, -0.5511789947129442, -1.8378490985067266, 0.1656381993108548, 1.2387742138438755, 0.9008788375077649]

tsdf_node:
  frame_id: task
  length: 0.3
  resolution: 40
  camera:
    frame_id: camera_depth_optical_frame
    info_topic: /camera/depth/camera_info
    depth_topic: /camera/depth/image_rect_raw

vgn_node:
  frame_id: task
  model: /home/michel/Developer/catkin_ws/src/vgn/assets/models/vgn_conv.pth
  finger_depth: 0.05
